app/controllers/chores_controller.rb

```diff
@@ -1,20 +1,49 @@
+require 'date'
```

Comment:
Good lesson. I was looking at official documentation and the sample included that `require` bit, so I assumed that `'date'` wasn't core to Ruby, but I didn't bother to check.